profile/Profile.proto

```diff
@@ -80,3 +81,17 @@ message ProfileTaskFinishReport {
     // profile task
     string taskId = 3;
 }
+
+// profile for go agent
+message GoProfileData{
+    string taskId = 1;
+    string traceSegmentId=2;
+    // the truly profile data
+    bytes payload=3;
+    // check last part true or false
+    bool isLast = 4;
+    // Span IDs with duration greater than minDuration,only in the last chunk
+    repeated int32 spanIds=5;
+    // use to reassemble fragments
+    int32 chunkId = 6;
```

Comment:
What does this field work for? You may send the profiling data without frequency, and sort by this field?